dask_geopandas/core.py

```diff
@@ -389,6 +390,41 @@ def morton_distance(self, p=15):
 
         return distances
 
+    def geohash(self, string=False, p=12):
```

Comment:
Shall we default to `False` here? I guess that most users will expect standard string-based geohash out of this method and we can set it to `False` easily manually in `spatial_shuffle`.

Reply:
that makes sense.